### DISCUSSION TITLE
Erase clips/snapshots which are not starred

### DISCUSSION THREAD

**Discussion by TheSleepySlee** [blakeblackshear/frigate#6747] [Ask A Question]
Hello,
I was wondering if there was a way to erase clips/snapshots that are not starred.
It seems possible to manually erase clips, but how can I erase multiple clips at the same time?

- For example erase all clips from a single camera?
- For example erase all clips BUT the ones starred

thanks

**Comment by NickM-27** [comment]
There is no way to delete multiple at the same time other than to delete one at a time. There is a pinned feature request for this
